### PR TITLE
Ditch proForma prefix from function names

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

File: contracts/truefi2/LoanFactory2.sol [hunk 1]
```diff
@@ -89,8 +89,8 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
     ) internal view returns (uint256) {
         uint8 borrowerScore = creditOracle.score(borrower);
         uint256 fixedTermLoanAdjustment = rateAdjuster.fixedTermLoanAdjustment(_term);
-        uint256 proFormaRate = rateAdjuster.proFormaRate(pool, borrowerScore, amount);
-        return proFormaRate.add(fixedTermLoanAdjustment);
+        uint256 Rate = rateAdjuster.Rate(pool, borrowerScore, amount);
```
Comment (barrutko) [inline contracts/truefi2/LoanFactory2.sol:92]: Why a capital letter?

File: contracts/truefi2/TrueFiPool2.sol [hunk 2]
```diff
@@ -629,15 +629,15 @@ contract TrueFiPool2 is ITrueFiPool2, IPauseableContract, ERC20, UpgradeableClai
 
     /**
      * @dev Ratio of liquid assets in the pool after lending
-     * @param amount Amount of asset being lent
+     * @param afterAmountLent Amount of asset being lent
      * @return Calculated ratio in basis points
      */
-    function proFormaLiquidRatio(uint256 amount) external override view returns (uint256) {
+    function LiquidRatio(uint256 afterAmountLent) external override view returns (uint256) {
```
Comment (yuchenlintt) [inline contracts/truefi2/TrueFiPool2.sol:635]: +1 to renaming `amount` to `afterAmountLent`. Nice improvement to readability.

File: contracts/truefi2/LoanFactory2.sol (hunk 1 above)
Comment (yuchenlintt) [reply contracts/truefi2/LoanFactory2.sol:92]: grep's too easy :)

Don't forget to uncapitalize all other function names too.

File: contracts/truefi2/TrueRateAdjuster.sol [hunk 3]
```diff
@@ -222,16 +222,16 @@ contract TrueRateAdjuster is ITrueRateAdjuster, UpgradeableClaimable {
         }
         uint256 coefficient = uint256(creditScoreRateConfig.coefficient);
         uint256 power = uint256(creditScoreRateConfig.power);
-        return min(coefficient.mul(MAX_CREDIT_SCORE**power).div(score**power).sub(coefficient), MAX_RATE_CAP);
+        return min(coefficient.mul(uint256(MAX_CREDIT_SCORE)**power).div(uint256(score)**power).sub(coefficient), MAX_RATE_CAP);
```
Comment (yuchenlintt) [inline contracts/truefi2/TrueRateAdjuster.sol:225]: I'm guessing this fixes the build warning? Thanks! Ideally we'd split this to a separate PR but I'll let it slide here.

**Review by yuchenlintt — CHANGES_REQUESTED** [review]

File: contracts/truefi2/interface/ITrueRateAdjuster.sol [hunk 4]
```diff
@@ -4,21 +4,21 @@ pragma solidity 0.6.10;
...
 
-    function proFormaPoolBasicRate(ITrueFiPool2 pool, uint256 amount) external view returns (uint256);
+    function poolBasicRate(ITrueFiPool2 pool, uint256 afterAmountLent) external view returns (uint256);
 
     function combinedRate(uint256 partialRate, uint256 __creditScoreAdjustmentRate) external pure returns (uint256);
 
     function creditScoreAdjustmentRate(uint8 score) external view returns (uint256);
 
-    function proFormaUtilizationAdjustmentRate(ITrueFiPool2 pool, uint256 amount) external view returns (uint256);
+    function utilizationAdjustmentRate(ITrueFiPool2 pool, uint256 amount) external view returns (uint256);
```
Comment (yuchenlintt) [inline contracts/truefi2/interface/ITrueRateAdjuster.sol:21]:
```suggestion
    function utilizationAdjustmentRate(ITrueFiPool2 pool, uint256 afterAmountLent) external view returns (uint256);
```

File: test/truefi2/TrueFiPool2.test.ts [hunk 5]
```diff
@@ -1034,7 +1034,7 @@ describe('TrueFiPool2', () => {
       })
     })
 
-    describe('proFormaLiquidRatio', () => {
+    describe('LiquidRatio', () => {
```
Comment (yuchenlintt) [inline test/truefi2/TrueFiPool2.test.ts:1037]:
```suggestion
    describe('liquidRatio', () => {
```

File: test/truefi2/lines-of-credit/TrueCreditAgency.test.ts [hunk 6]
```diff
@@ -1011,7 +1011,7 @@ describe('TrueCreditAgency', () => {
       await setUtilization(tusdPool, 50)
       const expectedCurrentRate = 693 // 300 + 100 + 143 + 150
       expect(await creditAgency.currentRate(tusdPool.address, borrower.address)).to.eq(expectedCurrentRate)
-      expect('proFormaRate').to.be.calledOnContractWith(rateAdjuster, [tusdPool.address, 223, 0])
+      expect('Rate').to.be.calledOnContractWith(rateAdjuster, [tusdPool.address, 223, 0])
```
Comment (yuchenlintt) [inline test/truefi2/lines-of-credit/TrueCreditAgency.test.ts:1014]:
```suggestion
      expect('rate').to.be.calledOnContractWith(rateAdjuster, [tusdPool.address, 223, 0])
```

File: test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts [hunk 7]
```diff
@@ -191,7 +191,7 @@ describe('TrueRateAdjuster', () => {
     })
   })
 
-  describe('proFormaRate', () => {
+  describe('Rate', () => {
```
Comment (yuchenlintt) [inline test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts:194]:
```suggestion
  describe('rate', () => {
```

File: test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts [hunk 8]
```diff
@@ -217,13 +217,13 @@ describe('TrueRateAdjuster', () => {
...
-      await mockPool.mock.proFormaLiquidRatio.withArgs(15_000).returns(10000 - 95 * 100)
+      await mockPool.mock.liquidRatio.withArgs(15_000).returns(10000 - 95 * 100)
       const expectedProFormaRate = 50000 // min(300 + 22600 + 7225 + 19950 = 50075, 50000)
-      expect(await rateAdjuster.proFormaRate(mockPool.address, borrowerScore, 15_000)).to.eq(expectedProFormaRate)
+      expect(await rateAdjuster.rate(mockPool.address, borrowerScore, 15_000)).to.eq(expectedProFormaRate)
     })
   })
 
-  describe('proFormaPoolBasicRate', () => {
+  describe('PoolBasicRate', () => {
```
Comment (yuchenlintt) [inline test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts:226]:
```suggestion
  describe('poolBasicRate', () => {
```

File: test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts [hunk 9]
```diff
@@ -269,7 +269,7 @@ describe('TrueRateAdjuster', () => {
     )
   })
 
-  describe('proFormaUtilizationAdjustmentRate', () => {
+  describe('UtilizationAdjustmentRate', () => {
```
Comment (yuchenlintt) [inline test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts:272]:
```suggestion
  describe('utilizationAdjustmentRate', () => {
```

**Review by yuchenlintt — APPROVED** [review]
LGTM, with a couple minor edits

**Review by yuchenlintt — DISMISSED** [review]

**Review by yuchenlintt — APPROVED** [review]

**Review by MiksuJak — APPROVED** [review]

**Pull request merged**